todo_app/templates/index.html

```diff
@@ -19,9 +19,26 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>Todo Items</h2>
       <ul class="list-group mb-4">
-        {% for item in items %}
+        {% for item in view_model.todo_items %}
+          <li>{{ item.title }} - {{ item.status}}</li>
+          <form action="/update_status" method="Post" class="form-example">
+            <div class="form-example">
+              <p hidden>
+                <lable for="name"> Todo Id: </lable>
+                <input type="text" name="todo-id" value="{{ item.id }}" required>
+              </p>
+            </div>
+            <div class="form-example">
+              <input type="submit" class="btn btn-success" value="Complete">
+            </div>
+          </form> 
+        {% endfor %}
+      </ul>
+      <h2>Done Items</h2>
+      <ul class="list-group mb-4">
+        {% for item in view_model.done_items %}
           <li>{{ item.title }} - {{ item.status}}</li>
           <form action="/update_status" method="Post" class="form-example">
```

Comment:
I don't think that this `update_status` form does anything for your done items, so is it worth removing it from this list of items?